package.json

```
@@ -43,33 +43,31 @@
 		"url": "https://opencollective.com/node-fetch"
 	},
 	"devDependencies": {
-		"@babel/cli": "^7.8.4",
-		"@babel/core": "^7.9.0",
-		"@babel/preset-env": "^7.9.5",
-		"@babel/register": "^7.9.0",
-		"@istanbuljs/nyc-config-babel": "^3.0.0",
 		"abort-controller": "^3.0.0",
 		"abortcontroller-polyfill": "^1.4.0",
-		"babel-plugin-add-module-exports": "^1.0.2",
-		"babel-plugin-istanbul": "^6.0.0",
+		"c8": "^7.1.2",
 		"chai": "^4.2.0",
 		"chai-as-promised": "^7.1.1",
 		"chai-iterator": "^3.0.2",
 		"chai-string": "^1.5.0",
-		"coveralls": "^3.0.13",
+		"coveralls": "^3.1.0",
+		"esm": "3.2.25",
```

Maybe add a caret here?

```suggestion
		"esm": "^3.2.25",
```